boms/integration-tests/src/test/java/com/google/cloud/MaximumLinkageErrorsTest.java

```diff
@@ -70,13 +76,24 @@ public void testMaximumLinkageErrors()
       ImmutableSet<ClassFile> currentReferences = currentProblems.get(problem);
       SetView<ClassFile> newReferences = Sets.difference(currentReferences, oldReferences);
       if (!newReferences.isEmpty()) {
-        StringBuilder message =
-            new StringBuilder("Newly introduced classes linking to " + problem + ":\n");
+        message.append("Newly introduced classes linking to " + problem + ":\n");
         for (ClassFile classFile : newReferences) {
           message.append("Link from " + classFile + "\n");
         }
         Assert.fail(message.toString());
       }
     }
   }
+
+  private String findLatestNonSnapshotVersion() throws MavenRepositoryException {
```

Comment:
There's already RepositoryUtility.findHighestVersion. I guess that returns snapshot versions too?

Reply:
Yes. Clarified it with comment.